terragrunt/aws/cloud_asset_inventory/cloudwatch_event.tf

```diff
@@ -3,7 +3,7 @@
 resource "aws_cloudwatch_event_rule" "cloudquery" {
   name                = "cloudquery"
   description         = "CloudQuery ECS Task"
-  schedule_expression = "cron(0 22 * * ? *)"
+  schedule_expression = "cron(0 4 ? * MON *)"
```

Comment:
Does this mean we'll only be getting an asset inventory refresh once a week or is this for something else?

Reply:
Yes, until we figure out if we want to continue with CQ with a paid subscription or change for another tool